packages/core/src/components/editable-text/editableText.tsx

```diff
@@ -99,7 +100,8 @@ export interface IEditableTextState {
     value?: string;
 }
 
-const BUFFER_WIDTH = 30;
+const BUFFER_WIDTH_EDGE = 5;
+const BUFFER_WIDTH_IE = 30;
```

Comment:
Yeah, they shift for different amounts. It'll "work" with both set to `30` as the buffer only needs to be a minimum, but looks better with the specific buffers.